### ISSUE TITLE
Files.TypeNameMatchesFileName - path 

### ISSUE THREAD

**Issue by wietsewarendorff** [slevomat/coding-standard#214]
Hi,

I have the codebase setup in a docker environment, all under the folder `/app/`. I have my psr-4 namespace setup as `"MyApp\\": "app/"` in my composer.json file, autoloading working as it should. But when I run the `SlevomatCodingStandard.Files.TypeNameMatchesFileName` tests,  all my files are reported, e.g.:
`Class name MyApp\Models\User does not match filepath  /app/app/Models/User.php.`

```
<rule ref="SlevomatCodingStandard.Files.TypeNameMatchesFileName">
    <properties>
        <property name="rootNamespaces" type="array" value="app=>MyApp"/>
    </properties>
</rule>
```

When I output the variables on [this line](https://github.com/slevomat/coding-standard/blob/master/SlevomatCodingStandard/Sniffs/Files/TypeNameMatchesFileNameSniff.php#L157), I see these results:

```
$typeName = 'MyApp\Models\User'
$expectedTypeName = 'MyApp\app\Models\User'
``` 

Seems to me the `$expectedTypeName` is incorrect (`app` should not be in there). This is probably caused by my folder structure and the fact that absolute paths are used to determine the namespace. Do you agree? And if so, is it possible to "detect" the expected namespace using relative paths only?

**Comment by kukulich** [comment]
@wietsewarendorff What's the absolute path of the file where `MyApp\Models\User` is located?

**Comment by wietsewarendorff** [comment]
@kukulich For a complete overview:
- root path of my project: `/app`
- location of vendor folder: `/app/vendor`
- UserModel: `/app/app/Models/User.php`

**Comment by kukulich** [comment]
@wietsewarendorff Try to add second property:

```
			<property name="skipDirs" type="array" value="
				app,
			"/>
```

**Comment by wietsewarendorff** [comment]
@kukulich this works for the example above, but I think this can cause issues when using other paths: if `/app/` is used somewhere, this solution will not work. Is it not possible to use the relative path?

**Comment by kukulich** [comment]
It looks it's not possible to determine current working directory from PHPCS so it's not possible to use relative path.
